cmds/tui.c

```diff
@@ -1514,7 +1558,7 @@ static struct tui_list * tui_session_init(struct opts *opts)
 		list_add_tail(&node->list, &tui_session.head);
 	}
 
-	tui_window_init(&tui_session.win, &session_ops);
+	tui_window_init(win, &session_ops);
```

Comment:
Please try to minimize your patch size.  Changes here and in `tui_info_init()` are not needed anymore.

Reply:
I'm not sure what the `minimize` the change in here?

Does the upper line refactor from `&tui_session.win` to `win` is unnecessary?
(for keeping consistency of code with first argument of `tui_window_init` with only `win` rather than `&<session>.win` 
at `tui_report_init`, `tui_session_init` and `tui_info_init`)

Or does the `tui_window_init` has to be removed?

Reply:
I mean the first one.  You don't need to create a local variable `win` just for single callsite IMO.